o8g/Extractor/config.xml

```diff
@@ -0,0 +1,76 @@
+﻿<xml>
+  <game cardsUrl="https://api.fiveringsdb.com/cards"
+        packsUrl="https://api.fiveringsdb.com/packs"
+        gameGuid="bb0f02e7-2a6f-4ae3-84a2-c501b4176844"
+        packId="id"
+        cardNumber="position"
+        cardOctgnNumber="Card Number"
+        cardName="name"
+        cardPackId="pack_code"
+        cardId="octgn_id"
+        cardImage="image_url"
+        />
+  <images>
+    <image name="FiveringsDB" url="{0}" />
+  </images>
+  <properties>
+    <property octgn_name="Clan" type="property" value="clan" capitalize="True" />
+    <property octgn_name="Text" type="property" value="text" isRich="True"/>
+    <property octgn_name="Type" type="property" value="type" capitalize="True" />
+    <property octgn_name="Traits" type="property" value="traits" capitalize="True" delimiter=". " />
```

Comment:
Traits normally end with the delimiter as well (I.e. single trait like Item.) Is it possible to have the trailing one as well? Not major but role proxy doesn't work atm since its pattern match was expecting it, that can be changed though

Reply:
This one was tricky cuz the json data writes the card traits as an actual list object, with no delimiters.  I might be able to add in the ending delimiter as a run, I’ll have to test it